### PR TITLE
Add MECHA token metadata

### PR DESCRIPTION
**Pull request by DoxxoSix**
# Add MECHA token metadata

Symbol: MECHA
Name: Mecha
Digits: 18
Aurora ERC-20 address:  0xa33C3B53694419824722C10D99ad7cB16Ea62754
URL: https://mechatracker.finance

### CODE REVIEW

File: assets/aurora.tokenlist.json [hunk 1]
```diff
@@ -436,6 +436,15 @@
...
     },
+    {
+      "chainId": 1313161554,
+      "address": "0xa33C3B53694419824722C10D99ad7cB16Ea62754",
+      "symbol": "MECHA",
+      "name": "Mecha Tracker",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/mecha.svg",
+      "tags": []
+    },
```
Comment (sept-en) [inline assets/aurora.tokenlist.json:447]: Tokens list shouldn't be manually updated.

```suggestion
```

File: tokens/mecha.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...3 76.8-37 16.7-37-16.7-37 16.7v-66.9l-37-16.7-185 83.6v41.8l37 16.8 111-50.3v33.5l27.7 12.6-27.7 12.5v41.8l37 16.8 185-83.7-.5-41.6z' fill='#b1f58e' data-original='#c6d1e0' %3E%3C/path%3E %3Cpath d='m252 93.5v33.5l37-16.7z' fill='#479222' data-original='#0f4c81' %3E%3C/path%3E %3Cpath d='m178 60v33.5l37-16.7z' fill='#479222' data-original='#0f4c81' %3E%3C/path%3E %3Cpath d='m178 93.5v41.8l27.7 12.6 9.3-4.2v-33.4z' fill='#479222' data-original='#0f4c81' %3E%3C/path%3E %3Cpath d='m178 160.4v41.8l37 16.8v-41.8z' fill='#479222' data-original='#0f4c81' %3E%3C/path%3E %3Cpath d='m30 93.5v41.8l37 16.8v-41.8z' fill='#479222' data-original='#0f4c81' %3E%3C/path%3E %3Cg fill='#597ba1'%3E %3Cpath d='m215 43.4 37 16.6v-33.4z' fill='#66cc33' data-original='#597ba1' %3E%3C/path%3E %3Cpath d='m215 76.8 37 16.7v-33.5z' fill='#66cc33' data-original='#597ba1' %3E%3C/path%3E %3Cpath d='m215 110.3v33.4l37-16.7v-33.5z' fill='#66cc33' data-original='#597ba1' %3E%3C/path%3E %3Cpath d='m215 177.2v41.8l185-83.7v-41.8z' fill='#66cc33' data-original='#597ba1' %3E%3C/path%3E %3Cpath d='m67 110.3v41.8l111-50.3v-41.8z' fill='#66cc33' data-original='#597ba1' %3E%3C/path%3E %3C/g%3E %3C/g%3E %3C/g%3E %3C/svg%3E",
```
Comment (sept-en) [inline tokens/mecha.json:7]:
```suggestion
  "icon": "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' version='1.1' xmlns:xlink='http://www.w3.org/1999/xlink' xmlns:svgjs='http://svgjs.com/svgjs' width='512' height='512' x='0' y='0' viewBox='0 0 430 430' style='enable-background: new 0 0 512 512' xml:space='preserve' class=''%0A%3E%3Cg transform='matrix(0.9999999999999999,0,0,0.9999999999999999,0,3.126388037344441e-13)' %3E%3Cg xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m178 337.5 37 16.7 37-16.7-37-16.7z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m215 287.3 37-16.7-37-16.7-185 83.6 37 16.7 111-50.1 37 16.7 37-16.7z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m363 320.8-37 16.7-37-16.7-37 16.7 37 16.7-111 50.2 37 16.8 185-83.7z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cg fill='%230f4c81'%3E%3Cpath d='m215 354.2 37-16.7-37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m289 387.7 37-16.7v-33.5l-37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m215 320.8 37-16.8-37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m363 320.8v33.4l37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m215 253.9v33.4l37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3C/g%3E%3Cpath d='m363 76.8-37 16.7-37-16.7-37 16.7v-66.9l-37-16.7-185 83.6v41.8l37 16.8 111-50.3v33.5l27.7 12.6-27.7 12.5v41.8l37 16.8 185-83.7-.5-41.6z' fill='%23b1f58e' data-original='%23c6d1e0' class='' %3E%3C/path%3E%3Cpath d='m252 93.5v33.5l37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m178 60v33.5l37-16.7z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m178 93.5v41.8l27.7 12.6 9.3-4.2v-33.4z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m178 160.4v41.8l37 16.8v-41.8z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cpath d='m30 93.5v41.8l37 16.8v-41.8z' fill='%23479222' data-original='%230f4c81' class='' %3E%3C/path%3E%3Cg fill='%23597ba1'%3E%3Cpath d='m215 43.4 37 16.6v-33.4z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m215 76.8 37 16.7v-33.5z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m215 110.3v33.4l37-16.7v-33.5z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m215 177.2v41.8l185-83.7v-41.8z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3Cpath d='m67 110.3v41.8l111-50.3v-41.8z' fill='%2366cc33' data-original='%23597ba1' class='' %3E%3C/path%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/svg%3E",
```

File: tokens/mecha.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "ethereum_address": "",
+  "aurora_address": "0xa33C3B53694419824722C10D99ad7cB16Ea62754",
+  "name": "Mecha Tracker",
```
Comment (sept-en) [inline tokens/mecha.json:4]: Token name should match metadata in ERC-20:

```suggestion
  "name": "Mecha",
```

**Review by sept-en — CHANGES_REQUESTED** [review]
Thanks for the PR! I added some comments.

File: assets/aurora.tokenlist.json (hunk 1 above)
Comment (DoxxoSix) [reply assets/aurora.tokenlist.json:447]: i dont understand. is there a non-manually way to add the entry? or it isnt necessary for the pr

File: tokens/mecha.json (hunk 2 above)
Comment (DoxxoSix) [reply tokens/mecha.json:7]: prettier style?

File: assets/aurora.tokenlist.json (hunk 1 above)
Comment (sept-en) [reply assets/aurora.tokenlist.json]: Yes, it will be done later with a tokens list generation tool.

**Review by sept-en — APPROVED** [review]

**Pull request merged**